wallet-new/src/Cardano/Wallet/Kernel/DB/Resolved.hs

```diff
@@ -56,6 +58,9 @@ data ResolvedTx = ResolvedTx {
 data ResolvedBlock = ResolvedBlock {
       -- | Transactions in the block
       _rbTxs  :: [ResolvedTx]
+
+      -- | Source MainBlock SlotId
```

Comment:
Don't really understand this comment. How about something like "The slot number of the slot this block appeared in"?